### ISSUE TITLE
Shortcuts menu is not keyboard accessible

### ISSUE THREAD

**Issue by micahgodbolt** [storybookjs/storybook#12819]
**Describe the bug**
When opening the shortcuts menu via mouse or keyboard, the items inside of the shortcuts menu are not accessible via keyboard.

**To Reproduce**

1. Go to https://next--storybookjs.netlify.app/official-storybook/
2. Click on "Shortcuts"
3. Press down arrow, or attempt to tab to "About your Storybook" link
4. Down arrow only scrolls down the page, and tabbing moves focus to search. There is no way to get to that (or any of the shortcut links) without using mouse

**Expected behavior**
As a dropdown, I would expect arrow keys to move up and down the menu

**Comment by tylerbutler** [comment]
Is the work on this still in progress? Anything we can do to raise priority or help?

**Comment by yannbf** [comment]
> Is the work on this still in progress? Anything we can do to raise priority or help?

There's a PR with a solution, hopefully will be done soon!

https://github.com/storybookjs/storybook/pull/12904#issuecomment-845886817

**Comment by vanessayuenn** [comment]
Closing as duplicate of #21578
